n3fit/src/n3fit/tests/test_evolven3fit.py

```diff
@@ -169,3 +169,29 @@ def test_perform_evolution(tmp_path, fitname):
     info = check_lhapdf_info(tmp_info)
     for datpath in tmp_nnfit.glob("replica_*/*.dat"):
         check_lhapdf_dat(datpath, info)
+
+
+@pytest.mark.parametrize("fitname", ["Basic_runcard_qed_3replicas_lowprec_398"])
```

Comment:
Instead of adding a complete new test you can put `Basic_runcard_qed_3replicas_lowprec_398` in the parametrize list of the test above. The test itself should do the same.